README.md

```diff
@@ -109,7 +109,8 @@ Server
 Usage: quicssh-rs server [OPTIONS]
 
 Options:
-  -l, --listen <LISTEN>  Address to listen on [default: 0.0.0.0:4433]
-  -h, --help             Print help
-  -V, --version          Print version
+  -l, --listen <LISTEN>        Address to listen on [default: 0.0.0.0:4433]
+  -p, --proxy-for <PROXY_FOR>  Address of the ssh server [default: 127.0.0.1:22]
```

Comment:
I prefer to use `proxy-to`. WDYM?

Reply:
at first I consulted dictionaries to say to use "proxy for" (of course it seems odd to me), and now I googled to find lots of example "proxy to <something like server>". I agree to you.